server/password-hash.js

```diff
@@ -1,6 +1,6 @@
 const passwordHashOld = require("password-hash");
 const bcrypt = require("bcryptjs");
-const saltRounds = 10;
+const saltRounds = bcrypt.genSaltSync();
```

Comment:
This would be incorrect behaviour.
See the docs of [`bcrypt.hashSync`](https://www.npmjs.com/package/bcryptjs#hashsyncs-salt) requires a number.
See the docs of [`bcrypt.genSaltSync`](https://www.npmjs.com/package/bcryptjs#gensaltsyncrounds-seed_length) returns a string

```suggestion
const saltRounds = 10;
```